### PR TITLE
Load balance picking epics

### PR DESCRIPTION
**Pull request by victorwuky**

### CODE REVIEW

File: product-development/README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ When picking a high priority unstarted epic, look at the epics _above the line_
 - My skill set is better suited for the third highest priority epic, so I'll pick that one, and save the first two for other folks who can finish them more quickly.
 - The fourth highest priority epic looks to be small so I'll pick that one first and finish it as a quick win before the end of the week.
 - My teammate has expertise in the highest priority epic and I'd like to collaborate and learn from them. But they aren't free just yet. I'll work on the third highest priority epic for now and plan to come back to pair with my teammate next week.
+- I see a few folks already working on Tilt Cloud epics, so I'll pick a Tilt epic to work on to try to load balance product iterations.
```
Comment (jazzdan) [inline product-development/README.md:43]: As of right now, with the exterminator rotation, we only have 4 full time engineers at most, sometimes only 3. Is it appropriate to state that as a goal we want people working on a diverse set of things when we also say that people should be working in **at least** pairs?

I'll note that this week everyone is working completely in silos as far as I am aware, with the possible exception of Han's work on the sample project with L.

I would rather have all 3/4 people working on the most important thing than have one person off doing less important work just to load balance product iterations. Especially in weeks where there are only 3 engineers around.

Comment (victorwuky) [reply product-development/README.md:43]: > I would rather have all 3/4 people working on the most important thing than have one person off doing less important work just to load balance product iterations. Especially in weeks where there are only 3 engineers around.

Thanks @jazzdan.  Agree we can’t really load balance and collaborate if we have so little people. Happy to close this PR. (After we’ve gotten some more feedback from Maia who brought up the topic.)

With Han close to wrapping up the sign in work (as they mentioned in the epics meeting), does it make sense for you and Matt to work on something together? I think this makes sense what you said regarding more collaboration and would love to see us driving toward that. And I like how you broke down the epics you’ve been working on and I’m guessing multiple people could work on it together?

Comment (jazzdan) [reply product-development/README.md:43]: Well there's still more work to do on this sign in stuff, I think it make more sense for @hyu and I to work together for the remainder of the week. Though that discussion probably doesn't belong here :)

Comment (victorwuky) [reply product-development/README.md:43]: Thanks for the update @jazzdan

**Comment by maiamcc** [comment]
Yeah I think that even tho my stated question was "how do we make sure we have a diverse set of epics", what I really meant to ask was "DO we want a diverse set of epics?" -- and that seems like a separate discussion. Happy to close this PR and discuss that second question elsewhere (or just have someone propose a decision that we'll jam on for a few weeks)

**Comment by victorwuky** [comment]
Thanks @maiamcc . I close this PR then. I think it would make sense to do a retro maybe in a few weeks to aggregate any successes / problems, after we've let this system run for a while. Thanks for bringing up this topic.

**Pull request closed without merging**